Tasks/PackerBuild/icon.svg

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
You might want to confirm if you are going to use the same icon as ARGDeployment task